src/backend/cuda/CMakeLists.txt

```diff
@@ -334,17 +334,23 @@ SET(libdevice_bc "")
 IF (USE_LIBDEVICE)
   SET(libdevice_computes "")
   LIST(APPEND libdevice_computes "20" "30" "35" "50")
+  LIST(APPEND cuda_root_list "${CUDA_NVVM_HOME}")
+  # Below location of libdevice files when installed via package manager on Ubuntu 16.04
+  LIST(APPEND cuda_root_list "/usr/lib/nvidia-cuda-toolkit")
```

Comment:
Doesn't cmake take care of slashes like the way it does for path hints we give for commands ?